modules/mod-deep-learning/DeepLearningAnalyzer.cpp

```diff
@@ -196,7 +196,10 @@ void DeepLearningAnalyzer::TensorToLabelTrack
    timestamps += tStart;
 
    // check that the output is one dimensional
-   wxASSERT(output.dim() == 1);
+   if (output.dim() != 1)
+      throw ModelException {
+         XO("Labeler model output is not a single track."), ""
```

Comment:
I changed it to the string "Model Error: The tensor output by the labeler model is not one dimension." just because it may be that the output is of dim 0 (i'm not sure why this would happen but the code doesn't distinguish between > 1 or 0)